src/components/Picklist/readme.md

```diff
@@ -317,3 +317,130 @@ initialState = { value: { name: 'option 3', label: 'Central Park' } };
     </GlobalHeader>
 </div>;
 ```
+
+##### Picklist with Modal
+
+```js
+import React from 'react';
+import { Picklist, Option, Modal, Input, Button, GoogleAddressLookup } from 'react-rainbow-components';
+import { Field, reduxForm } from 'redux-form';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBuilding } from '@fortawesome/free-solid-svg-icons';
+import styled from 'styled-components';
+
+initialState = { isOpen: false, selection: { name: '', label: '' } };
+
+const StyledField = styled(Field)`
+    margin-bottom: 30px;
+`;
+
+const StyledGoogleAddress = styled(GoogleAddressLookup)`
+    margin-bottom: 80px;
+`;
+
+const containerStyles = {
+    width: '220px'
+};
+
+function SimpleForm() {
+
+    const handleSubmit = values => {
+        console.log('submitted');
+    };
+
+    return (
+        <form id="redux-form-id" noValidate onSubmit={handleSubmit()}>
+            <StyledField
+                component={Input}
+                name="subject"
+                required
+                label="Building Name"
+                placeholder="Enter Building name"
+                icon={<BuildingIcon />}
+                className="rainbow-p-horizontal_xx-large"
+            />
+
+            <StyledGoogleAddress
+                id="gaddresslookup-1"
+                label="Building Location"
+                required
+                onChange={value => setState({ value })}
+                value={state.value}
+                placeholder="Enter the building Location"
+                apiKey={LIBRARY_GOOGLE_MAPS_APIKEY}
+                className="rainbow-p-horizontal_xx-large"
+            />
```

Comment:
This Input is not encapsulated within the Field of reduxForm. 

Should define the state well because it is not clear. In the examples we must make it as simple and easy to understand as possible